Description: Update path for wordlist and brutespray-output dirs
 This patch updates brutespray.py to use /usr/share/brutespray/.
Author: Stephane Neveu
Forwarded: not-needed
--- a/brutespray.py
+++ b/brutespray.py
@@ -251,7 +251,7 @@
 def brute(service,port,fname,output):
 
     if args.userlist is None and args.username is None:
-        userlist = 'wordlist/'+service+'/user'
+        userlist = '/usr/share/brutespray/wordlist/'+service+'/user'
         uarg = '-U'
     elif args.userlist:
         userlist = args.userlist
@@ -261,7 +261,7 @@
         uarg = '-u'
 
     if args.passlist is None and args.password is None:
-        passlist = 'wordlist/'+service+'/password'
+        passlist = '/usr/share/brutespray/wordlist/'+service+'/password'
         parg = '-P'
     elif args.passlist:
         passlist = args.passlist
